examples/step-55/doc/intro.dox

```diff
@@ -0,0 +1,141 @@
+<a name="Intro"></a> 
+<h1>Introduction</h1>
+
+<h3> Stokes Problem </h3>
+
+The purpose of this tutorial is to create an efficient linear solver
+for the Stokes equation and compare it to alternative
+approaches. Using FGMRES with geometric multigrid as a precondtioner
+for the velocity block, we see that the linear solvers used in Step-22
+cannot keep up since multigrid is the only way to get $O(n)$ solve
+time. Using the Timer class, we collect some statistics to compare
+setup times, solve times, and number of iterations. We also compute
+errors to make sure that what we have implemented is correct.
+
+Let $u \in H_0^1 = \{ u \in H^1(\Omega), u|_{\partial \Omega} = 0 \}$
+and $p \in L_*^2 = \{ p_f \in L^2(\Omega), \int_\Omega p_f = 0
+\}$. The Stokes equations read as follows in non-dimensionalized form:
+
+@f{eqnarray*} - 2 \text{div} \frac {1}{2} \left[ (\nabla \textbf{u}) +
+(\nabla \textbf{u})^T\right] + \nabla p & =& f \\ - \nabla \cdot u &=&
+0 @f}
+
+Note that we are using the deformation tensor instead of $\Delta u$ (a
+detailed desription of the difference between the two can be found in
+Step-22, but in summary, the deformation tensor is more physical as
+well as more expensive).
+
+<h3> Linear solver and preconditioning issues </h3> The weak form of
```

Comment:
This leads to funny formatting -- you need to have the header on a line by itself.